app/Console/Commands/RemoveDuplicatePermissionsCommand.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Permission;
+use Illuminate\Console\Command;
+
+class RemoveDuplicatePermissionsCommand extends Command
+{
+    protected $signature = 'utrs-maintenance:remove-duplicate-permissions 
+                            {--dry : A dry run of the command should be ran}';
```

Comment:
Disagree with the removal of the '?'. Dry is an optional command and the braces don't make it optional.
https://laravel.com/docs/6.x/artisan#arguments

Reply:
'?' is not needed for flags:

```
C:\Users\taavi\sites\utrs>php artisan utrs-maintenance:remove-duplicate-permissions

 You are about to run a live database change. Are you sure? (Did you test it?) (yes/no) [no]:
 > no

Operation aborted.

C:\Users\taavi\sites\utrs>php artisan utrs-maintenance:remove-duplicate-permissions --dry
No duplicates found.
```

Reply:
Oh I guess it's seen as an option in the next section on that laravel doc page which doesn't require it...interesting.